content/kubermatic/main/tutorials-howtos/administration/presets/_index.en.md

```diff
@@ -48,29 +44,35 @@ Preset list offers multiple options that allow Administrators to manage Presets.
 1. A list of providers configured for the Preset.
 
 
-## Creating a Preset
+### Creating a Preset
 
 Open the `Create Preset` dialog through the button available on the Preset list.
 
 ![](/img/kubermatic/main/ui/create_preset.png?height=300px&classes=shadow,border)
 
-### 1. Preset Step
+#### Step 1: Preset
 
 ![](/img/kubermatic/main/ui/create_preset_first_step.png?height=500px&classes=shadow,border)
 
-- `Name` is a required parameter and will be used in the wizard to select the provider
-- `Domain` is an optional parameter that allows to limit Preset availability only to the specific users with email matching this domain
-- `Hide upon creation` allows to hide the preset after creation from the users
+- `Name` is a required parameter and will be used in the wizard to select the provider.
+- `Domain` is an optional parameter that allows to limit Preset availability only to the specific users with email matching this domain.
+- `Projects` is an optional list of projects that allows to limit Preset availability to the specific projects.
+- `Hide upon creation` allows to hide the preset after creation from the users.
```

Comment:
What does it mean for a preset to be hidden? Isn't its purpose to be used by users, how could they do that if the preset is hidden?